build failure with latest nightly

rustc 1.28.0-nightly (990d8aa74 2018-05-25)
I was trying to build rustfmt-nightly v0.8.2

````
   Compiling rustc-ap-rustc_data_structures v147.0.0
error[E0599]: no method named `start_bound` found for type `R` in the current scope
   --> /home/matthias/.cargo/registry/src/github.com-1ecc6299db9ec823/rustc-ap-rustc_data_structures-147.0.0/array_vec.rs:122:33
    |
122 |         let start = match range.start_bound() {
    |                                 ^^^^^^^^^^^
error[E0599]: no method named `end_bound` found for type `R` in the current scope
   --> /home/matthias/.cargo/registry/src/github.com-1ecc6299db9ec823/rustc-ap-rustc_data_structures-147.0.0/array_vec.rs:127:31
    |
127 |         let end = match range.end_bound() {
    |                               ^^^^^^^^^
error[E0599]: no method named `start_bound` found for type `R` in the current scope
   --> /home/matthias/.cargo/registry/src/github.com-1ecc6299db9ec823/rustc-ap-rustc_data_structures-147.0.0/sorted_map.rs:218:33
    |
218 |         let start = match range.start_bound() {
    |                                 ^^^^^^^^^^^
error[E0599]: no method named `end_bound` found for type `R` in the current scope
   --> /home/matthias/.cargo/registry/src/github.com-1ecc6299db9ec823/rustc-ap-rustc_data_structures-147.0.0/sorted_map.rs:233:31
    |
233 |         let end = match range.end_bound() {
    |                               ^^^^^^^^^
error: aborting due to 4 previous errors
````

Perhaps bumping the dependency version of the package fixes the problem?

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -47,8 +47,8 @@ env_logger = "0.5"
 getopts = "0.2"
 derive-new = "0.5"
 cargo_metadata = "0.5.1"
-rustc-ap-rustc_target = "147.0.0"
-rustc-ap-syntax = "147.0.0"
+rustc-ap-rustc_target = "148.0.0"
+rustc-ap-syntax = "148.0.0"
 failure = "0.1.1"
 
 [dev-dependencies]

--- a/src/expr.rs
+++ b/src/expr.rs
@@ -336,8 +336,6 @@ pub fn format_expr(
                 ))
             }
         }
-        // FIXME(#2743)
-        ast::ExprKind::ObsoleteInPlace(..) => unimplemented!(),
     };
 
     expr_rw

--- a/src/test/mod.rs
+++ b/src/test/mod.rs
@@ -900,6 +900,10 @@ fn rustfmt() -> PathBuf {
     me.pop(); // chop of the test name
     me.pop(); // chop off `deps`
     me.push("rustfmt");
+    assert!(
+        me.is_file() || me.with_extension("exe").is_file(),
+        "no rustfmt bin, try running `cargo build` before testing"
+    );
     return me;
 }
 

